workspace/app/services/dev-settings.svc.js

```diff
@@ -9,11 +9,11 @@ angular.module('impacWorkspace').service('DevSettings', function ($q, ImpacRoute
   var DEFAULTS = {
     // API Endpoints
     mnoeUrl: {
-      host: 'https://impac-mnoe-uat.maestrano.io',
+      host: 'http://localhost:7000',
```

Comment:
Hey @rheasunshine can you please remove this from the commit.. just squash and push -f

Reply:
done